superset-frontend/src/components/Table/cell-renderers/NumericCell/index.tsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/naming-convention */
```

Comment:
I added this single override, as it felt sensible to leave the locales and currencies as-is